drivers/espi/CMakeLists.txt

```diff
@@ -2,3 +2,5 @@
 
 zephyr_library()
 
+zephyr_library_sources_ifdef(CONFIG_ESPI_XEC		espi_mchp_xec.c)
```

Comment:
Having the name of the manufacturer in the driver filename is not mandatory. But if you want it, you will have to write it all like "espi_microchip_xec.c"

But espi_xec.c would be totally fine as well

Reply:
Agree, but we already agreed this as file naming for Microchip drivers

Reply:
ok missed that. fine by me